fixtures/FakeNet/methods/getUTXO.js

```diff
@@ -0,0 +1,38 @@
+const fs = require('fs');
+const { is } = require('../../../src/utils');
+
+function getUtxoOfAddressAtHeight(address, height) {
+  let utxos = [];
+  try {
+    const utxofile = JSON.parse(fs.readFileSync(`./fixtures/FakeNet/data/utxos/${address}.json`));
+    for (let i = parseInt(Object.keys(utxofile)[0], 10) - 1; i <= height; i++) {
+      const el = utxofile[i];
+      if (el) {
+        utxos = utxos.concat(el);
+      }
+    }
+  } catch (e) {
+    // s
```

Comment:
Hiding errors is a bad idea.

Reply:
Agreed, here is the thing : 
If there is a file locally matching addresses we returns the UTXO at height there. 
If not, then we return empty array. The error is FS error. It will throw a lot, can do an instanceof here to handle the differences between file not exist and other issues.

Reply:
Maybe just check if a file exists with `fs.existsSync(path)`?

Reply:
Absolutely ! Fixed :)